test/manual/kerberos.test.js

```diff
@@ -51,7 +51,16 @@ describe('Kerberos', function () {
     });
   });
 
-  // Unskip this test when a proper setup is available - see NODE-3060
+  it('validate that CANONICALIZE_HOST_NAME can be passed in', function (done) {
+    const client = new MongoClient(
+      `${krb5Uri}&authMechanismProperties=SERVICE_NAME:mongodb,CANONICALIZE_HOST_NAME:true&maxPoolSize=1`
+    );
+    client.connect(function (err, client) {
+      expect(err).to.not.exist;
```

Comment:
I know this is in line with the rest of the tests, but I'm wondering if it would be better to do `if (err) { return done(err) }` instead, or even better, specify this as an async function and avoid the callback nesting altogether

Reply:
we may need to keep the done callback in the `it` function definition, since right now the verifyKerberosAuthentication execution won't call done and it's not written as a promise; if that's too much of a headache or too weird to rewrite, maybe we just keep the old syntax for the test